lib/semantics/resolve-names.cc

```diff
@@ -2459,6 +2531,22 @@ bool DeclarationVisitor::OkToAddComponent(
   }
 }
 
+// ConstructNamesVisitor implementation
+
+bool ConstructNamesVisitor::CheckDef(const std::optional<parser::Name> &x) {
```

Comment:
Is there a reason that this function has a `bool` result, given that it's always true?  Will it be extended in the future with code that could return false?

Reply:
No, I think it's just a historical artifact. I'm doing more work on constructs, so I'll clean this up in a future change.